distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Table_Helpers.enso

```diff
@@ -223,7 +225,9 @@ type Table_Column_Helper
         just_indicators = table_with_blank_indicators.select_columns (blanks.map .name) on_problems=Problem_Behavior.Report_Error
 
         # Maximum is equivalent to Exists and Minimum is equivalent to Forall.
-        col_aggregate = if when_any then Aggregate_Column.Maximum _ else Aggregate_Column.Minimum _
+        col_aggregate = case when of
+            Blank_Selector.Any_Cell_Blank -> Aggregate_Column.Maximum
+            Blank_Selector.All_Cells_Blank -> Aggregate_Column.Minimum
```

Comment:
Does this work without the `_` underscores??

IIRC without the underscore this uses default values and creates a `Aggregate_Column.Maximum 0` which will always check the _first_ column. That will inevitably be incorrect once `columns.length > 1`.

Do the tests still succeed with that? If yes, I think we will need to expand the tests to ensure such an error is caught to avoid regressions.